pkg/process/net/common_darwin.go

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build darwin
+
+package net
+
+import (
+	"fmt"
+
+	model "github.com/DataDog/agent-payload/v5/process"
+	sysconfig "github.com/DataDog/datadog-agent/cmd/system-probe/config"
+)
+
+const (
+	statsURL             = "http://unix/debug/stats"
+	registerURL          = "http://unix/" + string(sysconfig.NetworkTracerModule) + "/register"
+	tracerouteURL        = "http://unix/" + string(sysconfig.TracerouteModule) + "/traceroute/"
+	languageDetectionURL = "http://unix/" + string(sysconfig.LanguageDetectionModule) + "/detect"
+	pprofURL             = "http://unix/debug/pprof"
+
+	// pingURL is not used in windows, the value is added to avoid compilation error in windows
+	pingURL = "http://unix/" + string(sysconfig.PingModule) + "/ping/"
```

Comment:
Should we only register tracerouteURL or also provide some of the other common urls?